### ISSUE TITLE
Okta sign in widget text does not have sufficient color contrast

### ISSUE THREAD

**Issue by christopherhuii** [okta/okta-signin-widget#2062]
### I'm submitting a
  - [x] bug report
  - [ ] feature request

### Background info

I noticed that links in the Okta Sign In Widget do not have sufficient color contrast. I noticed this specifically with the `Need help signing in?` and `Forgot password?` links.

Those links are `#777777` (`rgb(119, 119, 119`). The background is `#ffffff`. It does not have sufficient [color contrast](https://webaim.org/resources/contrastchecker/?fcolor=777777&bcolor=FFFFFF) to satisfy WCAG AA.

**There might be other places that don't have sufficient color contrast.** It's worth investigating.

### Expected behavior

I would expect the default sign in widget styles to be at a minimum WCAG AA compliant.

### What went wrong?

Text does not have sufficient color contrast

### Your environment

  - Okta Sign-In Widget Version: `v5.7.1`
  - Browser: Chrome
  - OS: macOS
  - Language: Javascript/React

**Comment by bryanapellanes-okta** [comment]
@christopherhuii,
Thanks for reaching out.  I'll bring this to the attention of our Javascript developers and someone will follow up.

Thanks for using Okta!
